content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>To avoid problems with users of different OSs, I suggest using <code>os.path</code> here:</p><p> </p><pre class="ql-syntax" spellcheck="false">import os
import imageio

DIR = 'tutorial-x-ray-image-processing'

xray_image = imageio.imread(os.path.join(DIR, '00000011_001.png'))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
That's a good idea. Add the OS module 👍